pkg/meta/model/job_test.go

```diff
@@ -293,7 +293,6 @@ func TestJobCodec(t *testing.T) {
 		ID:         16,
 		Type:       ActionFlashbackCluster,
 		BinlogInfo: &HistoryInfo{},
-		Args:       []any{0, map[string]any{}, "ON", true},
```

Comment:
does it matter if kept?

Reply:
The order of arguments here is wrong, and they're not used. So I just removed them.